lib/teracy-dev-k8s/processors/settings.rb

```diff
@@ -99,6 +88,11 @@ def generate_nodes(k8s_config)
         network_type = k8s_config['network']['type']
         subnet = k8s_config['network']['subnet']
         os = k8s_config['os']
+        supported_os = k8s_config['supported_oses'].select { |index| (index['_id'] == os ) ? index : nil }.first
```

Comment:
index => item, index is not a proper name here

Reply:
supported_os = k8s_config['supported_oses'].find { |item| item['_id'] == os  }

see: https://ruby-doc.org/core-2.2.3/Enumerable.html#method-i-find